nordicsemi/dfu/dfu_transport_ble.py

```diff
@@ -60,6 +60,10 @@
 global nrf_sd_ble_api_ver
 nrf_sd_ble_api_ver = config.sd_api_ver_get()
 
+import platform
+if platform.system().lower() == "linux":
+    from bluez import BluezBleAdapter
```

Comment:
Shouldn't it be `from nordicsemi.dfu.bluez import BluezBleAdapter`

Reply:
good catch